core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -51,6 +52,16 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       "-disown"
     )
 
+    conf.sslConf match {
+      case Some(ssl) => {
+        cmdToLaunch ++ Array("-internal_security_config", ssl)
```

Comment:
This would ensure that just communication is secured between h2o cluster which we are starting on yarn and sparkling water is secured.

We need to also handle the case where the h2o cluster is already started ( locally, anywhere ) and we are connection to it from sparkling water. But it seems like any h2o cluster started with the configuration bellow should use secure communication, right ? I'm just making sure here all cases are covered